packages/cli/src/commands/dev.ts

```diff
@@ -328,6 +330,77 @@ async function refreshEndpoint(apiClient: TriggerApi, endpointId: string, endpoi
   }
 }
 
+async function getOutdatedTriggerDevPackages(path: string) {
+  const packageJSONPath = pathModule.join(path, "package.json");
+  const packageJSON = await readJSONFile(packageJSONPath);
+
+  const packageJSONSchema = z.object({
+    dependencies: z.record(z.string()),
+    devDependencies: z.record(z.string()),
```

Comment:
`devDependencies` should be optional. The default with a new nextjs app is none

Reply:
Cool. Noted!